gui/serial_unix.cc

```diff
@@ -21,6 +21,68 @@
 #include "mainwindow.h"
 #if !defined (Q_OS_MAC) // FIXME: find a better way to hide this on Mac.
 
+#ifdef HAVE_UDEV
+#include <libudev.h>
+#include <QDebug>
+
+static QStringList dynamicDevices()
+{
+    struct udev *udev = udev_new();
+    if(!udev) {
+        qDebug() << "Can't create udev";
+        return QStringList();
+    }
+
+    QSet<QString> devices;
+
+    struct udev_enumerate *enumerate = udev_enumerate_new(udev);
+    udev_enumerate_add_match_subsystem(enumerate, "tty");
```

Comment:
Will this get the bluetooth and USB/serial devices? Seems like it should, but it's perhaps an edge case.